contrib/mesos/pkg/executor/executor.go

```diff
@@ -179,6 +166,10 @@ func New(config Config) *KubernetesExecutor {
 	return k
 }
 
+func (k *KubernetesExecutor) InitialRegComplete() <-chan struct{} {
```

Comment:
nit: can we just call this something simpler like `Registration()` and add a doc that says how to use the chan? e.g. `closes upon completion of the executor's Registered callback`